### PR TITLE
API for setting QoS priority of data channels

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-pc/issues/955

### CODE REVIEW

**Comment by aboba** [comment]
@pthatcherg Can you review?

File: webrtc.html [hunk 1]
```diff
@@ -7762,6 +7778,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              USVString      protocol = "";
              boolean        negotiated = false;
              unsigned short id;
+             RTCPriorityType priority;
```
Comment (alvestrand) [inline webrtc.html]: If we add a default value of "low" here, we don't have to special-case the missing attribute in the handling instructions above.

RTCPriorityType priority = "low";

(Note: I can't tell what the default for RTP channels is from the document. "low" sounds right, but consistency is important.)

**Comment by alvestrand** [comment]
Looks good to me (modulo the issue of how to specify default priority).

**Comment by aboba** [comment]
@alvestrand Added the default priority of "low".

**Pull request merged**